tests/backup/backup_resiliency_test.go

```diff
@@ -1272,21 +1277,25 @@ var _ = Describe("{RebootNodesWhenBackupsAreInProgress}", func() {
 			log.Infof("Switching cluster context to destination cluster")
 			err := SetDestinationKubeConfig()
 			log.FailOnError(err, "Switching context to destination cluster failed")
-			listOfWorkerNodes = node.GetWorkerNodes()
+			//listOfStorageDriverNodes = node.GetStorageDriverNodes()
 			for i := 0; i < 2; i++ {
```

Comment:
How about len(listOfWorkerNodes) instead of 2?

Reply:
need to reboot only 2 nodes not all

Reply:
Got it.